src/ucp/core/ucp_context.h

```diff
@@ -136,6 +136,8 @@ struct ucp_config {
     char                                   *env_prefix;
     /** MD to compare for transport selection scores */
     char                                   *selection_cmp;
+    /** Register whole mem_type allocation */
+    int                                    mem_type_reg_whole_alloc;
```

Comment:
pls add it to ucp_context_config_t instead, then no need to add to ucp_context as well